src/main/osd/osd.c

```diff
@@ -91,6 +91,16 @@
 #include "hardware_revision.h"
 #endif
 
+#ifndef CORE_TEMPERATURE_THRESHOLD
+// a temperature above 70C should produce a warning, lockups have been reported above 80C.
```

Comment:
This should be a general comment about how `CORE_TEMPERATURE_THRESHOLD` works.

Reply:
done